platform/admin/ui/src/main/webapp/js/models/Installer.js

```diff
@@ -136,17 +136,17 @@ define([
                     url: that.installUrl + 'admin-post-install-modules/true',
                     dataType: 'JSON'
                 }).then(function(){
-                    if (reboot) {
+                    if (shutdown) {
                         $.ajax({
                             type: 'GET',
-                            url: that.rebootUrl,
+                            url: that.shutdownUrl,
                             dataType: 'JSON'
                         }).done(function () {
                             window.setTimeout(function () {
                               window.location.href = that.get("redirectUrl");
                             }, 30000);
                         }).fail(function () {
-                            alert("Error trying to restart system. Please manually restart the system.");
+                            alert("Error trying to shutdown system. Please manually shutdown the system.");
```

Comment:
I don't know what we should display but this isn't going to be an error this will be success